me.hyliu.fluentreader.yml

```diff
@@ -0,0 +1,90 @@
+app-id: me.hyliu.fluentreader
+runtime: org.freedesktop.Platform
+runtime-version: "20.08"
+branch: stable
+sdk: org.freedesktop.Sdk
+base: org.electronjs.Electron2.BaseApp
+base-version: "20.08"
+sdk-extensions:
+  - org.freedesktop.Sdk.Extension.node14
+command: start-fluent-reader
+separate-locales: false
+finish-args:
+  - --share=ipc
+  - --socket=x11
+  - --device=dri
+  - --socket=pulseaudio
+  - --share=network
+modules:
+  # First step is to install Node to /app/node, that way it can be accessible outside of the sdk
+  # environment. install-sdk.sh is used because install.sh omits npm.
+  # (This does not need to be done for electron-builder, see
+  # electron-webpack-quick-start's build.electron.webpack.ElectronWebpackQuickStart.yaml for an
+  # explanation.)
+  # - name: node
+  #   buildsystem: simple
+  #   build-commands:
+  #     - '/usr/lib/sdk/node14/install-sdk.sh'
+
+  # Now is the quickstart module.
+  - name: fluent-reader
+    buildsystem: simple
+    build-options:
+      # Add the node bin directory.
+      append-path: /usr/lib/sdk/node14/bin
+      env:
+        # Set the Electron cache directory.
+        # (The directory format is: /run/build/MODULE_NAME/flatpak-node/electron-cache)
+        # ELECTRON_CACHE: '/run/build/fluent-reader/flatpak-node/electron-cache'
+        XDG_CACHE_HOME: /run/build/fluent-reader/flatpak-node/cache
+        # Sets the directory where Node is located so way npm won't download the headers.
+        # npm_config_nodedir: '/usr/lib/sdk/node14'
+    build-commands:
+      # Have Yarn use the offline mirror.
+      - "npm install --offline --prefix=fluent-reader --cache=$FLATPAK_BUILDER_BUILDDIR/flatpak-node/npm-cache"
+      # Download the packages.
+      # If you were using npm with electron-webpack/electron-builder, then the above two commands
+      # would look more like the npm commands in the vanilla-quick-start manifest, just without
+      # the --prefix.
+
+      # Run electron-builder, passing the architecture arguments to it.
+      # Note that the -- is important; without that, the argument will be passed to
+      # yarn instead of electron-builder.
+      - ". flatpak-node/electron-builder-arch-args.sh; npm run --prefix=fluent-reader --offline build; npm run --prefix=fluent-reader --offline package-linux-unpacked -- $ELECTRON_BUILDER_ARCH_ARGS"
```

Comment:
You can use multiline strings in YAML to make this much more readable.

Reply:
Done